modules/tinymce/changelog.txt

```diff
@@ -8,6 +8,8 @@ Version 5.5.0 (TBD)
     Added `fullscreen_native` setting to the fullscreen plugin to enable use of the entire monitor #TINY-6284
     Added the `origin` property to the `ObjectResized` and `ObjectResizeStart` events, to specify which handle the resize was preformed on #TINY-6242
     Added new StyleSheetLoader `unload` and `unloadAll` APIs to allow loaded stylesheets to be removed #TINY-3926
+    Added the `lineheight` format, and `LineHeight` query command and action to the editor #TINY-4843
```

Comment:
Yeah, dropping the format probably makes sense. Otherwise, even just switching the order (e.g. format and command/action) reads a little better... It is hard to word this though